prov/efa/src/rxr/rxr.h

```diff
@@ -175,6 +175,11 @@ static inline void rxr_poison_mem_region(uint32_t *ptr, size_t size)
  */
 #define RXR_LONGCTS_PROTOCOL BIT_ULL(8)
 
+/* Flag to tell
```

Comment:
weird line wrap location.

Shouldn't the flag really be to mark a packet as a copy?  That seems like the special case (given all the ways we do zero copy sends for non-eager messages).

Reply:
Fixed line wrap. Also changed the flag to indicate data has been copied for send

Reply:
Removed this patch in new revision.